bridge/test/NFTBridge_test.js

```diff
@@ -117,5 +120,99 @@ contract("Bridge NFT", async function(accounts) {
         });
       });
     });
+
+      describe("createSideNFTToken", async function () {
+
+          let symbol;
+          let name;
+          let baseURI;
+          let tokenAddress;
+
+          beforeEach(async function() {
+              symbol = await this.token.symbol();
+              name = await this.token.name();
+              baseURI = await this.token.baseURI();
+              tokenAddress = this.token.address;

```

Comment:
As all this values are inmutable, maybe it's better to convert this beforeEach to before

Reply:
Actually the token contract is created on a beforeEach call from before (I think it makes sense to recreate the contract for each test to avoid a dirty context, right?) - so if we keep that, since this depends on it, we'd have to keep it this way

Reply:
you are right, didn't notice the contract was on a beforeEach